experiments/task.py

```diff
@@ -117,13 +112,13 @@ def setup(self):
             rwd = block["rwd"]
 
             for i in range(n):
-                c1 = c2 =  np.searchsorted(P_cue, random.uniform(0,1))
+                c1 = c2 =  np.searchsorted(P_cue, np.random.uniform(0,1))
```

Comment:
From the last time I tested it, the np.random is rather slow for a unique scalar. This was the reason to use the python random generator instead.